fabtests/functional/unexpected_msg.c

```diff
@@ -318,6 +318,7 @@ int main(int argc, char **argv)
 
 	opts = INIT_OPTS;
 	opts.iterations = 600; // Change default from 1000.
+	opts.transfer_size = 128;
```

Comment:
Note that PR #8920 removes the tcp restrictions

Reply:
Do you want me to remove the changes here and wait for #8920? I already have to rebase to pick up the EFA changes in #8935

Reply:
No need -- we can keep these for now.  I have other changes to this test out of tree that I'll eventually push upstream after these are merged.